core/src/com/serwylo/beatgame/levels/Level.kt

```diff
@@ -171,7 +227,7 @@ object TheOriginalWorld: World {
             TheExerciseRoom,
             Vivaldi,
             ReorientTheReceivingAntenna,
-            CustomLevel,
+            LegacyCustomLevel,
```

Comment:
Do we still need this here? Shouldn't it get migrated to the new custom world implementation?